routing/endpointregistry.go

```diff
@@ -86,13 +86,14 @@ func (r *EndpointRegistry) Do(routes []*Route) []*Route {
 
 	for _, route := range routes {
 		if route.BackendType == eskip.LBBackend {
-			for _, epi := range route.LBEndpoints {
-				metrics := r.GetMetrics(epi.Host)
-				if metrics.DetectedTime().IsZero() {
-					metrics.SetDetected(now)
+			for i := range route.LBEndpoints {
```

Comment:
It should update timestamps for network backends as well, see https://github.com/zalando/skipper/pull/2759/files#r1441546084